lambda/health_package/tox.ini

```diff
@@ -4,7 +4,7 @@ envlist =
 skipsdist = true
 
 [flake8]
-max-line-length = 88
+max-line-length = 92
```

Comment:
I think this was set to match the default configuration from `black`. Do you need to change it?

Reply:
88 pls

Reply:
Done